app/models/concerns/proposal_delegate.rb

```diff
@@ -40,16 +40,20 @@ module ProposalDelegate
 
     ###################################################
 
-    def self.client
```

Comment:
I would rather see `def self.client_slug` for consistency with what it returns: a short string identifying the customer namespace.